### PR TITLE
Proposal: import statement

### PR DESCRIPTION
**Pull request by Arthurdw**

### CODE REVIEW

File: concepts/imports.coral [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+    ),
+
+    // Normal module import
+    "coesmoes/seo",
+
+    // Module and relative path imports with one alias
+    "math",
+    "./utils/math" as utils,
+
+    // Root path import
```
Comment (Sculas) [inline concepts/imports.coral]:
```suggestion
    // Import from project root
```

File: concepts/imports.coral [hunk 2]
```diff
@@ -0,0 +1,17 @@
+import (
+    // Multiple submodule imports
+    "org/example" (
+        "components.MyCustomComponent",
```
Comment (Sculas) [inline concepts/imports.coral]:
```suggestion
        // Static import
        "components.MyCustomComponent",
```

**Review by Sculas — CHANGES_REQUESTED** [review]
Got some suggestions, besides that LGTM.
Maybe modify the logic concept file with current proposal?

File: concepts/imports.coral (hunk 1 above)
Comment (Sculas) [reply concepts/imports.coral]: This one is really up to you. "Root path" just doesn't seem clear to me.

**Review by Sculas — APPROVED** [review]

File: concepts/logic.coral [hunk 3]
```diff
@@ -5,6 +5,10 @@ Logic components are defined by using the 'ssr' or 'logic' keyword. A logic
 component ALWAYS expects a return value of the Component type.
 **/
 
+<!-- CoralForm is a form component that prevents default form behavior -->
```
Comment (Sculas) [inline concepts/logic.coral]:
```suggestion
// CoralForm is a form component that prevents default form behavior
```
**HTML comments should not be mixed with Coral comments.**

**Comment by ThimoPlays** [comment]
Looks good for now, might need some changes once we have more clarity regarding (local) package management.

**Review by ThimoPlays — APPROVED** [review]

**Review by Sculas — CHANGES_REQUESTED** [review]

File: concepts/imports.coral [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+
+    // Normal module import
+    "coesmoes/seo",
+
+    // Module and relative path imports with one alias
+    "math",
+    "./utils/math" as utils,
+
+    // Import from project root
+    "/components/header"
```
Comment (Sculas) [inline concepts/imports.coral]:
```suggestion
    "/components.Header"
```
I think you intended this?

**Review by Sculas — CHANGES_REQUESTED** [review]

File: concepts/logic.coral (hunk 3 above)
Comment (Arthurdw) [reply concepts/logic.coral]: Oops, this was because of some quick copy-pasting. 😅

File: concepts/imports.coral (hunk 4 above)
Comment (Arthurdw) [reply concepts/imports.coral]: Already got resolved, thanks though. ♥

**Review by Sculas — APPROVED** [review]

**Pull request merged**